config/vufind/config.ini

```diff
@@ -765,13 +765,22 @@ url_shortener_key_type = md5
 [Database]
 ; Connection string format is [platform]://[username]:[password]@[host]:[port]/[db]
 ; where:
-; [platform] = database platform (mysql, oci8 or pgsql)
+; [platform/driver] = database platform (mysql, oci8 or pgsql)
 ; [username] = username for connection
 ; [password] = password for connection (optional)
 ; [host] = host of database server
 ; [port] = port of database server (optional)
 ; [db] = database name
+; Use of database is deprecated, use individual parameters instead
```

Comment:
I don't think we should deprecate this as part of the present pull request, since we'll need to change some other things (at a minimum: the installation process, the Phing build.xml configuration) to favor the new method before we can fully deprecate it. Maybe for now, the comment could be:

```
; For more granular configuration, comment out the database setting and use the individual parameters below.
```

...and then we can change it to a deprecation warning in a separate PR that also addresses those other things.